env/env_posix.cc

```diff
@@ -488,6 +488,7 @@ Env* Env::Default() {
   CompressionContextCache::InitSingleton();
   INIT_SYNC_POINT_SINGLETONS();
   // ~PosixEnv must be called on exit
+  //**TODO: Can we make this a STATIC_AVOID_DESTRUCTION?
```

Comment:
I don't think so, but I don't know all the details of shutting down background threads.